tasks/main.yml

```diff
@@ -21,9 +21,9 @@
   when: not elasticsearch_installed.stat.exists
 
 - name: Install ES plugins
-  shell: /usr/share/elasticsearch/bin/plugin {% if elasticsearch_file_name|get_major_version == 1 %}--{% endif %}remove {{ item }} && /usr/share/elasticsearch/bin/plugin {% if elasticsearch_file_name|get_major_version == 1 %}--{% endif %}install {{ item }}
-  when: head_plugin_path.stat.exists != true
+  shell: /usr/share/elasticsearch/bin/plugin {% if elasticsearch_file_name|get_major_version == 1 %}--{% endif %}install {{ item }}
   with_items: "{{ plugins_to_install }}"
+  ignore_errors: True
```

Comment:
The error message is pretty short. 

```
ERROR: plugin directory /usr/share/elasticsearch/plugins/head already exists. To update the plugin, uninstall it first using 'remove head' command
```

Unfortunately we don't know the directory name so we can't use all of that. I don't think its that important to totally nail this down, plugins can just be installed if anything goes wrong. As long as we can run the playbook multiple times without it crashing I think its ok?